src/daft-parquet/src/read.rs

```diff
@@ -363,7 +363,6 @@ async fn read_parquet_single(
 async fn stream_parquet_single(
     uri: String,
     columns: Option<&[&str]>,
-    start_offset: Option<usize>,
```

Comment:
start_offset is never used for swordfish, which does not rely on unloaded micropartitions.